examples/03_connectivity/plot_seed_to_voxel_correlation.py

```diff
@@ -172,7 +172,7 @@
 # interest.
 from nilearn import plotting
 
-display = plotting.plot_stat_map(seed_based_correlation_img, threshold=0.3,
+display = plotting.plot_stat_map(seed_based_correlations_fisher_z, threshold=0.3,
```

Comment:
because a correlation of 0.3 seems substantial (but beware estimators with low degrees of freedom). By contrast, a z value of 0.3 is very low, basically chance level. You probably want z values greater than 2. This being said, I realize that nothing survives at a threshold of 2.
A threshold of 1 only outlines the PCC.